test/distributed/test_c10d_common.py

```diff
@@ -1116,6 +1116,32 @@ def _test_warn_not_in_group(self, backend):
             dist.barrier(group=group)
             dist.broadcast(x, src=0, group=group)
 
+    def _test_rank_membership(self, backend):
```

Comment:
Can we do anything to simulate behavior in a multi-node scenario?

Reply:
This change is not multi-node related, I guess this confusion is the same as the previous one.